### ISSUE TITLE
you do not see the images, but only the outline without the image

### ISSUE THREAD

**Issue by rgianpaolo** [jrwieland/Heliwizard-taranis#1]
Hi, I copied the file in the Wizard folder as described, but in create new model the images are not visible, only the empty frames are seen, without image.

**Comment by jrwieland** [comment]
Did you just copy the wizard.lua file?   If so all of the images cannot be located. The images are located in the zip file.    I placed this out here until it is added into openTX 2.3.15.  I am working on the companion new model wizard, but it is taking a bit to make something cross platform.  Here are the images you need. 
extract them to your /SCRIPTS/WIZARD folder
[img.zip](https://github.com/jrwieland/Heliwizard-taranis/files/6912069/img.zip)
Let me know if you have any other issues.

**Comment by rgianpaolo** [comment]
Hi, many thanks.
I Copy all file in to the folder. This is the result. There are four empty boxes.
Best regards
Gianpaolo
--
Inviato da myMail per Android domenica, 01 agosto 2021, 04:48AM +02:00 da John Wieland  ***@***.*** :

**Comment by rgianpaolo** [comment]
[1627801454233](https://user-images.githubusercontent.com/13065456/127762391-917cf1dc-be70-497c-a5ae-6a71344565ef.jpg) like to sen![1627801454233](https://user-images.githubusercontent.com/13065456/127762391-917cf1dc-be70-497c-a5ae-6a71344565ef.jpg)

**Comment by jrwieland** [comment]
I put it on my old 9x radio this morning copying the image files and wizard.lua to the SD card.

 it worked in both creating a new model.

Here is the result
![IMG_20210801_071513784.jpg](https://user-images.githubusercontent.com/20526716/127768918-f18cd4fb-6074-4578-b97f-1e41afceb19c.jpg)

In scripts wizard you should see the following:
![IMG_20210801_072258889.jpg](https://user-images.githubusercontent.com/20526716/127769058-e0fcaf4f-3423-44ec-8237-c12f6676c415.jpg)

**Comment by jrwieland** [comment]
@rgianpaolo let me know if this is resolved
I suspect that the img folder was not copied to the folder.

**Comment by rgianpaolo** [comment]
Ok Ok Ok, perfect !!!!! I found out what I was doing wrong, I'm sorry. I deleted all the images contained in the Wizard folder of the radio SD card. Then I just copied all the images and copied them in full in the wizard folder. Instead I had to copy the folder you sent me in full without transferring the images only. I understood all this from the second image you sent me. Thank you so much. Continue your wonderful and very useful work.
